src/screens/VisualTests/VisualTests.tsx

```diff
@@ -376,12 +383,11 @@ export const VisualTests = ({
     // We always want to show the status of the running build (until it is done)
     (runningBuild && runningBuild.step !== "complete") ||
     // Even if there's no build running, we want to show the next build if it hasn't been selected.
-    (nextBuild && nextBuild.id !== storyBuild?.id);
+    (canSwitchToNextBuild && nextBuild.id !== storyBuild?.id);
   const buildStatus = showBuildStatus && (
     <BuildProgress
       runningBuild={runningBuild}
-      nextBuild={nextBuild}
-      switchToNextBuild={switchToNextBuild}
+      switchToNextBuild={canSwitchToNextBuild && switchToNextBuild}
```

Comment:
I don't think `canSwitchToNextBuild` is needed here, since `switchToNextBuild` also checks it.

Reply:
The `BuildProgress` component uses it to tell if it should display the link or not.

See:
 - https://www.chromatic.com/test?appId=6480e1b0042842f149cfd74c&id=64f567c8beb8a0bf7bd418de vs
 - https://www.chromatic.com/test?appId=6480e1b0042842f149cfd74c&id=64f567c8beb8a0bf7bd418df